content/en/blog/2023/rust-based-ztunnel/index.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Introducing Rust-Based Ztunnel for Istio Ambient Service Mesh"
+description: A purpose-built per-node proxy for Istio ambient mesh.
+publishdate: 2023-01-30
+attribution: "Lin Sun (Solo.io), John Howard (Google)"
+keywords: [istio,ambient,ztunnel]
+---
+
+The ztunnel (zero trust tunnel) component is a purpose-built per-node proxy for Istio ambient mesh. It is responsible for securely connecting and authenticating workloads within ambient mesh. Ztunnel is designed to focus on a small set of features for your workloads in ambient mesh such as mTLS, authentication, L4 authorization and telemetry, without terminating workload HTTP traffic or parsing workload HTTP headers. The ztunnel ensures traffic is efficiently and securely transported to the waypoint proxies, where the full suite of Istio's functionality, such as HTTP telemetry and load balancing, is implemented.
+
+Because ztunnel is designed to run on all of your Kubernetes worker nodes, it is critical to keep its resource footprint small. ztunnel is designed to be an invisible (or "ambient") part of your service mesh with minimal impact to your workloads.
```

Comment:
```suggestion
Because ztunnel is designed to run on all of your Kubernetes worker nodes, it is critical to keep its resource footprint small. Ztunnel is designed to be an invisible (or "ambient") part of your service mesh with minimal impact to your workloads.
```